ui/packages/catalog-ui-search/src/main/webapp/react-component/workspaces-templates/presentation.tsx

```diff
@@ -17,7 +17,7 @@ import styled from '../styles/styled-components'
 import { CustomElement } from '../styles/mixins'
 import { ChangeBackground } from '../styles/mixins'
 import { transparentize, readableColor } from 'polished'
-import Text from '../container/input-wrappers/text'
+import Text from '../input-wrappers/text'
```

Comment:
That's actually not what's going on here. `input-wrappers` is a special directory that's holding all the React-wrapped Marionette components we're trying to move away from. This import is actually coming from `../input-wrappers/text/index.js` not a named export. I kept them this way for consistency. I think exporting from named files would be fine too, but I don't think we should worry too much about it because ideally we delete `input-wrappers` soon.